lib/puppet/provider/package/gem.rb

```diff
@@ -1,8 +1,9 @@
 require 'puppet/provider/package'
```

Comment:
Is this still needed with the new base class?

Reply:
No!